test/distributed/org/apache/cassandra/distributed/test/DistributedReadWritePathTest.java

```diff
@@ -0,0 +1,272 @@
+package org.apache.cassandra.distributed.test;
+
+import org.apache.cassandra.distributed.api.ConsistencyLevel;
+import org.apache.cassandra.distributed.api.ICluster;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.apache.cassandra.distributed.api.Feature.NETWORK;
+
+// TODO: this test should be removed after running in-jvm dtests is set up via the shared API repository
+public class DistributedReadWritePathTest extends TestBaseImpl
```

Comment:
this looks like you are based off old trunk, you are adding back something Benedict renamed...

Reply:
Right; that got preserved the right one.